PermissionsScraper/PermissionsScraper/Triggers/PermissionsReverseLookupTableGenerator.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+using GitHubContentUtility.Common;
+using GitHubContentUtility.Operations;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using PermissionsScraper.Common;
+using PermissionsScraper.Models;
+using PermissionsScraper.Services;
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using GitHubRepoAppConfig = GitHubContentUtility.Common.ApplicationConfig;
+using PermissionsAppConfig = PermissionsScraper.Common.ApplicationConfig;
+
+namespace PermissionsScraper.Triggers
+{
+    public static class PermissionsReverseLookupTableGenerator
+    {
+        private const string PermissionsDocumentFile = "PermissionsDocumentFile";
+        private const string ReverseLookupTable = "ReverseLookupTable";
+
+        [FunctionName("PermissionsReverseLookupTableGenerator")]
+        public static void Run([TimerTrigger("%ScheduleTriggerTime%")] TimerInfo myTimer, ILogger logger)
```

Comment:
Do we plan for this function to execute the same time as the `DescriptionScraper`?